examples/applications/object_detection/retina_net/basic/pascal_voc/loss.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+
+class FocalLoss(tf.keras.losses.Loss):
```

Comment:
This should be renamed to something like `RetinaNetLoss`. Focal Loss is on line 79.